src/components/ebay-menu-button/examples/19-prefix-label/template.marko

```diff
@@ -0,0 +1,8 @@
+<ebay-menu-button type="radio" prefix-label="Select From Menu">
+    <@label>
+        label text
```

Comment:
Like in the listbox button example, can you make sure the label text is updated to reflect the current selection.